website/docs/docs/deploy/model-notifications.md

```diff
@@ -3,37 +3,30 @@ title: "Model notifications"
 description: "While a job is running, receive email notifications in real time about any issues with your models and tests. "
 ---
 
-# Model notifications <Lifecycle status="beta" />
-
 Set up dbt to notify the appropriate model owners through email about issues as soon as they occur, while the job is still running. Model owners can specify which statuses to receive notifications about: 
 
 - `Success` and `Fails` for models
 - `Warning`, `Success`, and `Fails` for tests
 
 With model-level notifications, model owners can be the first ones to know about issues before anyone else (like the stakeholders). 
 
-:::info Beta feature
-
-This feature is currently available in [beta](/docs/dbt-versions/product-lifecycles#dbt-cloud) to a limited group of users and is gradually being rolled out. If you're in the beta, please contact the Support team at support@getdbt.com for assistance or questions.
-
-:::
-
 To be timely and keep the number of notifications to a reasonable amount when multiple models or tests trigger them, dbt observes the following guidelines when notifying the owners:  
 
-- Send a notification to each unique owner/email during a job run about any models (with status of failure/success) or tests (with status of warning/failure/success). Each owner receives only one notification, the initial one.
-- Don't send any notifications about subsequent models or tests while a dbt job is still running.
-- At the end of a job run, each owner receives a notification, for each of the statuses they specified to be notified about, with a list of models and tests that have that status. 
+- Each owner/user who subscribes to notifications for one or more statuses (like failure, success, warning) will receive only _one_ email notification at the end of the job run.
+- The email includes a consolidated list of all models or tests that match the statuses the user subscribed to, instead of sending separate emails for each status.
+- Notifications about subsequent models or tests aren't sent while a dbt job is still running. Users are only notified once the job has fully completed.
```

Comment:
@mirnawong1 we still want to describe what happens during a run, so could we do something like this - 
```
- Send a notification to each unique owner/email during a job run about any models (with status of failure/success) or tests (with status of warning/failure/success). Each owner receives only one notification, the initial one.
- Don't send any notifications about subsequent models or tests while a dbt job is still running.
- Each owner/user who subscribes to notifications for one or more statuses (like failure, success, warning) will receive only _one_ email notification at the end of the job run.
- The email includes a consolidated list of all models or tests that match the statuses the user subscribed to, instead of sending separate emails for each status.
```